terraform/install.sh

```diff
@@ -150,15 +215,15 @@ installTerraform
 #export GOOGLE_APPLICATION_CREDENTIALS=""
 #gcloud auth application-default login
 
-# Ensure no google.com accounts early - they are not supported!
 acct=$(gcloud info --format="value(config.account)")
 if [[ $acct == *"google.com"* ]];
 then
-  log "Google.com accounts are currently not supported by Stackdriver Sandbox.";
-  exit;
+  folder_id="262044416022"            
 fi;
 
 # Provision Stackdriver Sandbox cluster
+getProject;
+createProject;
```

Comment:
Having createProject here is a bit misleading, since a project is only created sometimes. Consider removing `create_flag` and putting `createProject` inside the logic of `getProject` instead

Reply:
Ok. I'll do that.